packages/mpt/package.json

```diff
@@ -8,9 +8,9 @@
     "trie",
     "ethereum"
   ],
-  "homepage": "https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/trie#readme",
+  "homepage": "https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/mpt#readme",
   "bugs": {
-    "url": "https://github.com/ethereumjs/ethereumjs-monorepo/issues?q=is%3Aissue+label%3A%22package%3A+trie%22"
+    "url": "https://github.com/ethereumjs/ethereumjs-monorepo/issues?q=is%3Aissue+label%3A%22package%3A+mpt%22"
```

Comment:
Ah ok, we should remove the `package: trie` label (or at least add `package: mpt` label) and update our general issue template to also point to the mpt label and not the trie label.